moped-editor/src/routes.js

```diff
@@ -121,3 +121,5 @@ export const restrictRoutes = (routes) =>
       return protectedRoute;
     }
   });
+
+export const restrictedRoutes = restrictRoutes(routes);
```

Comment:
this does make more sense to export it like this, instead of exporting routes and the function and making the `restrictedRoutes` in App.js, but is there a performance benefit to doing it like this?

Reply:
yes—it just seemed like a red flag to have the routing object at the very root of our app be re-constructed on every App render. i doubt it's a noticeable performance improvement, but these little performance leaks do accumulate over time 🤷‍♂️